README.md

```diff
@@ -10,18 +11,30 @@ $$ T(n) =
     \end{cases}
 $$
 
-2.
+$T(n/13^i) + 5i$ <br/>
+From $13^i = n$ we get $i = log_{13}(n)$ <br/>
+Plugging i back in we get that $T(n) \in \Theta(log(n))$ <br/>
+
+3.
 $$ T(n) =
     \begin{cases}
         1 & n \leq 1\\
         13 T\left(\frac{n}{13}\right) + 5 & n > 1
     \end{cases}
 $$
 
-3.
+$13^i * T(n/13^i) + ($ $\sum_{j=0}^{i-1} 13^j)$ <br/>
+From $13^i = n$ we get $i = log_{13}(n)$ <br/>
+Plugging i back in we get that $T(n) \in \Theta(n))$ <br/>
+
+4.
 $$ T(n) =
     \begin{cases}
         1 & n \leq 1\\
         13 T\left(\frac{n}{13}\right) + 2n & n > 1
     \end{cases}
 $$
+
+$13^i * T(n/13^i) + (($ $\sum_{j=0}^{i-1} 13^j) * n)$ <br/>
+From $13^i = n$ we get $i = log_{13}(n)$ <br/>
+Plugging i back in we get that $T(n) \in \Theta(n))$ <br/>
```

Comment:
This one is incorrect -- is it really not doing any more work than the previous one?